core/templates/pages/story-editor-page/story-editor-page.component.ts

```diff
@@ -43,23 +45,24 @@ require('pages/story-editor-page/story-editor-page.constants.ajs.ts');
 require('services/bottom-navbar-status.service.ts');
 require('services/page-title.service.ts');
 require('services/loader.service.ts');
+require('services/ngb-modal.service.ts');
 require('services/prevent-page-unload-event.service.ts');
 
 import { Subscription } from 'rxjs';
```

Comment:
Move this import above `SavePendingChangesModalComponent` import

Reply:
DONE